script/cibuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```

Comment:
I don't understand why we added the config arguments, I think @greggersh would have to answer that